Add Spell Scroll Cards to deckbuilding (#850)
Heegu-sama/Homm3BG: sections/deckbuilding.tex

Whenever you play an Ability or Statistic Card, you must choose which effect you are using.
The number of \svg{expert} Effects you can use each Round is limited by your Main Hero's \pagelink{Level}{Level}.
Track the number of uses you have in any suitable manner, such as by moving Black Cubes on and off your Hero Card.\par
\vspace*{\fill}
\columnbreak

\begin{expansion}[before=\vspace*{-11mm}]{navalbattles}
  \subsection*{\pagetarget{Empowered Ability}{Empowered} Ability Cards}\index{Empowered Statistic Card}
  These Cards are more powerful versions of the normal Ability Cards.
  They have only one effect which is identical to the normal Ability's Expert effect, but does not require using your \svg{expert}.

  \medskip

>   These cards can be gained by spending an \pagelink{Empowered Ability Token}{Empowered Ability Token}.

Reviewer: ```suggestion
  These Cards can be gained by spending an \pagelink{Empowered Ability Token}{Empowered Ability Token}.
```

      \draw (0.75, 1.9) node {\encircle{1}};
      \draw (-1, -1.5) node {\encircle{2}};
    \end{tikzpicture}
    \end{scriptsize}
    \textbf{\textit{\textcolor{darkcandyapplered}{Empowered Statistic Card}}}
  \end{center}
  \begin{itemize}
    \item[\textbf{4.}] Necropolis-specific Card
  \end{itemize}
  \vspace*{\fill}
\end{multicols}

\begin{tikzpicture}[remember picture, overlay]
  \node[rotate=-28, opacity=0.15] at ([xshift=0em, yshift=-8em]current page.center) {\includegraphics[width=0.5\linewidth]{\art/frost_ring.png}};
\end{tikzpicture}
\begin{multicols*}{2}

\vspace*{\fill}
\note{11}{
  Certain Cards are limited to the Necropolis Faction \svg{necro-yellow}.
  When a non-Necropolis player draws a Necropolis Card\index{Necropolis Card} from the Ability Deck, they may either discard it and draw a new Card as a replacement or gain it.
  Non-Necropolis players \textbf{cannot use} Faction Specific Cards from their hand in any way besides for effects that discard them.
}
\vspace*{\fill}
\columnbreak

\vspace*{\fill}
\begin{expansion}[before=\vspace*{-2em}]{inferno}
  \subsection*{\pagetarget{Empowered Statistic}{Empowered} Statistic Cards}\index{Empowered Statistic Card}

>   The Empowered Statistic Cards work very similar to Empowered Ability Cards.

Reviewer: ```suggestion
  The Empowered Statistic Cards work very similarly to Empowered Ability Cards.
```

>   Their only effect corresponds to the Expert effect of normal Statistic Cards, but without requiring to use your \svg{expert}.

Reviewer: ```suggestion
  Their only effect corresponds to the Expert effect of normal Statistic Cards, but without requiring you to use your \svg{expert}.
```

  \vspace*{\fill}
\end{center}

\pagebreak

\begin{expansion}{stronghold}
\subsection*{\pagetarget{Spell Scroll Cards}{Spell Scroll Cards}}

During setup shuffle 10 of the Spell Scroll Cards into the Artifact Deck.
Set the remaining 10 Cards aside to form a Spell Scroll Deck.
Some locations or Scenario effects may call for them later.
\vspace*{1em}

{
    \medskip
    \centering
    \includegraphics[width=0.7\linewidth]{\cards/test_spellscroll.png}\\
    \medskip
    \footnotesize
    \begin{center}
         \textbf{\textit{\textcolor{darkcandyapplered}{Spell Scroll Card}}}
    \end{center}
}

\vspace*{1em}

> When you gain a Spell Scroll place it near your Hero Card, take the 2 top Cards from the Spell Deck and place them face down on the Spell Scroll.

Reviewer: ```suggestion
When you gain a Spell Scroll, place it next to your Hero Card, take the 2 top Cards from the Spell Deck and place them face-down on the Spell Scroll.
```

> You can look at these Spells at any time and they don't count for your Hand Limit.

Reviewer: ```suggestion
You can look at these Spells at any time and they don't count toward your Hand \svg{hand} Limit.
```

You can only have 2 Spell Scrolls at a time.
If you gain a third one, draw the new Spells normally and then decide whether you discard the newly gained Spell Scroll or replace one you already have (including all Spells on it).

\medskip

You can play Spells from a Spell Scroll like any other Spell with the following exceptions:
\begin{itemize}
    \item They don't count towards your Spell Limit.
    You can also play a Spell from your hand during this Combat Round.

>     \item You can only use their lowest \svg{empower} effect.

Reviewer: I'm not a fan of "lowest" here, as the bottom (low) effect on the card is the strongest one. Maybe cheapest? Weakest? Compare it to the original, where it's "lowest power":
<img width="787" height="201" alt="image" src="https://github.com/user-attachments/assets/2da3cc59-f5a7-43d3-bc3e-9fe5c156531b" />
